### ISSUE TITLE
Getting vector::_M_range_check Error

### ISSUE THREAD

**Issue by Sahkoojp** [zrax/pycdc#356]
I'm trying to decompile 3.11 pyc and I'm getting this:
```
# Source Generated with Decompyle++
# File: "name" (Python 3.11)

Error decompyling "name": vector::_M_range_check: __n (which is 70) >= this->size() (which is 66)
```
Any help? Ps I'm still noob so sorry if it's a noob error.
Edit: .pyc file size is 34kb.

**Comment by endigma** [comment]
Getting this too, but with `vector::_M_range_check: __n (which is 10) >= this->size() (which is 10)`. File size is 5.6k, if that matters

**Comment by Imma-P** [comment]
Me too: 
vector::_M_range_check: __n (which is 20) >= this->size() (which is 14)

**Comment by avatarlegend** [comment]
help please me too

**Comment by zrax** [comment]
Should be fixed with #462
